package-lock.json

```diff
@@ -10766,4 +10766,4 @@
       "dev": true
     }
   }
-}
+}
```

Comment:
Should have a new line at the end of the file.

Reply:
Done